lib/fog/digitalocean/requests/compute_v2/list_flavors.rb

```diff
@@ -2,18 +2,18 @@ module Fog
   module Compute
     class DigitalOceanV2
       class Real
-        def list_flavors
+        def list_flavors(filters = {})
           request(
             :expects => [200],
             :method  => 'GET',
-            :path    => '/v2/sizes'
+            :path    => "/v2/sizes#{filters.to_a.map { |x| "#{x[0]}=#{x[1]}" }.join("&")}"
```

Comment:
I agree, that's definitely something I'll do when I consolidate the calls. Is `CGI.escape` the preferred library in fog for URL-escaping input?